pkg/scheduler/factory/factory.go

```diff
@@ -149,10 +149,12 @@ type Configurator interface {
 	GetPredicateMetadataProducer() (predicates.PredicateMetadataProducer, error)
 	GetPredicates(predicateKeys sets.String) (map[string]predicates.FitPredicate, error)
 
-	// Needs to be exposed for things like integration tests where we want to make fake nodes.
-	GetNodeLister() corelisters.NodeLister
```

Comment:
move these fields to line155 is not necessary to me.

Reply:
Leaving to the approver.